lib/linguist/vendor.yml

```diff
@@ -41,6 +41,9 @@
 # Node dependencies
 - node_modules/
 
+# Yarn releases
+- .yarn/releases
```

Comment:
This is a regular expression, so the dot needs to be escaped to avoid matching something like `eyarn/releases` or what-have-you.

In addition, this will also target files named `releases` that simply reside in a directory whose name *ends* with `yarn`, such as:

~~~ignorelist
/node-yarn/releases
~~~

I recommend replacing this with:

~~~yaml
# Yarn releases
- (^|/)\.yarn/releases/
~~~

(Note that the other patterns in this file have similar mistakes; this was only noticed [very recently](https://github.com/github/linguist/pull/4776#pullrequestreview-344970330), hence why they're not fixed).

Reply:
```suggestion
- (^|/)\.yarn/releases/
```

Reply:
Great, thanks for your suggestions @Alhadis! I've updated accordingly